bot/commands/recommend.py

```diff
@@ -35,54 +36,46 @@ def recommend_output(self, record: models.ContentRecord) -> Embed:
         )
         return em  # TODO: add header/footer images with users pfp
 
-    @commands.group(name="recommend")
-    async def recommend(self, ctx: UtilityContext) -> None:
-        if ctx.invoked_subcommand is None:
-            await ctx.send_help()
-
-    @recommend.command(name="anime")
-    async def recommend_anime(
-        self, ctx: UtilityContext, member: Member, *, name: str
+    @commands.command(name="recommend")
+    async def recommend(
+        self,
+        ctx: UtilityContext,
+        content_type: ContentType,
+        member: Member,
+        *,
+        name: str,
     ) -> None:
-        """Recommend an anime to another user."""
+        """Recommend content to users"""
         async with ctx.typing():
-            data = await anime.get_anime_manga(
-                self.bot, query=name, _type=ContentType.Anime
+            if content_type == ContentType.Anime:
+                data = await anime.get_anime_manga(
+                    self.bot, query=name, _type=ContentType.Anime
+                )
+                name = data["title"]
+                url = data["siteUrl"]
+            elif content_type == ContentType.Manga:
+                data = await anime.get_anime_manga(
+                    self.bot, query=name, _type=ContentType.Manga
+                )
+                name = data["title"]
+                url = data["siteUrl"]
```

Comment:
We can get away with directly passing in the `content_type` for both these cases
```suggestion
            if content_type in [ContentType.Anime, ContentType.Manga]:
                data = await anime.get_anime_manga(
                    self.bot, query=name, _type=content_type
                )
                name = data["title"]
                url = data["siteUrl"]
```